src/si/mod.rs

```diff
@@ -49,6 +49,7 @@ system! {
         temperature_interval::TemperatureInterval,
         thermodynamic_temperature::ThermodynamicTemperature,
         time::Time,
+        torque_magnitude::TorqueMagnitude,
```

Comment:
I'm inclined towards `Torque` instead of `TorqueMagnitude`. See `Force` as an existing example. Additionally I'd like to get support for mathematical vectors as the underlying storage type some day (e.g. `Quantity<V = Vec3f32>` rather than `Vec3f32<T = Quantity>`). Thoughts?

Reply:
I've done the renaming.  In terms of planning for vectors, I'll spend some time thinking about it and let you know what would be a best case scenario for my uses.

Reply:
So, I spent some time poking around and it seems like nobody has written the vector library I want, yet.  It's very tempting to launch yet another hobby project.  What I really want is compile time checking of transform matrix math, to prevent nonsense matrix combinations (ie if Tab transforms a point or vector from a to b, I want `Tab*Tbc == Tac` and both `Tab*Tcb` and `Tbc*Tab` should break the build)... but I don't strictly need them to be quantities, all I really care about is compile time frame of reference checking.

I'm curious though, what's your motivation for wanting a vector quantity (as opposed to a vectory of quantities)?

Reply:
I figured a `Quantity<V = Vec3f32>` would be more natural to use than `Vec3<T = Quantity>`. No actual usage experience and I'm not looking to attempt to implement now unless someone actually has a use case.